osi_common.proto

```diff
@@ -191,13 +194,13 @@ message MountingPosition
 // 
 // Azimuth and elevation are defined as the rotations that would have to be 
 // applied to the local frame (e.g sensor frame definition in 
-// \c OSI:DetectionHeader) to make its x-axis point towards the referenced point
+// \c DetectionHeader) to make its x-axis point towards the referenced point
```

Comment:
SensorDetectionHeader?

Reply:
@CarloVanDriestenBMW I changed all DetectionHeader to SensorDetecionHeader.

Reply:
Thx